cov_profiles.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Fetches coverage data for each fuzz test, combines them, and produces a
+# coverage profile that can be analyzed.
+if [[ "$#" -ne 1 ]]; then
+    echo "Usage: $0 LND_DIR"
+    exit 1
+fi
+
+readonly LND_DIR=$1
+readonly CACHE_DIR=$(mktemp -d)
+
+# Get the directory of cov_profiles.sh in case it is being called not in
+# lnd-fuzz.
+BASE_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+packages=("lnwire" "brontide" "htlcswitch/hop" "tlv" "watchtower/wtwire" "watchtower/wtclient" "zpay32") 
+
+# collect_combine_cov_profiles collects coverage for each fuzzing package and
+# then combines them.
+function collect_combine_cov_profiles {
+    coverage_dirs=()
+
+    # Collect coverage profiles.
+    for p in ${packages[@]}; do
+        cd $PWD/$p/testdata/fuzz
+
+        for f in $(ls $PWD); do
+            # Move corpus to CACHE_DIR.
+            mkdir -p $CACHE_DIR/$f
+            cp -a "${BASE_DIR}/$p/testdata/fuzz/${f}"/ "${CACHE_DIR}/$f/"
+            num_inputs=$(ls "${CACHE_DIR}/$f/" | wc -l | xargs)
```

Comment:
Can we drop the `xargs`?

Reply:
nope

Reply:
Why?  The merging script doesn't have `xargs`.

Reply:
The last time I ran it the merging script didn't work on macOS w/o some modifications